cypress/fixtures/index.css

```diff
@@ -21,10 +21,9 @@ article figure img {
   column-gap: 0px;
 }
 
-#photos-grid img {
-  /* Just in case there are inline attributes */
-  width: 100% !important;
-  height: auto !important;
+img[ix-sizes='auto'] {
+  display: block;
+  width: 100%;
```

Comment:
This rule not only helps [reduce layout shifts](https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/) but it also tells our module what the available width for the image will be before it's been loaded. We need this information in order to avoid always defaulting the parent elements width when setting `sizes`.